### PR TITLE
[Tests] Update test-pwm.js for bug fix

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

File: tests/test-pwm.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ expectThrow("pwmpin: set pulseWidth without period", function () {
     pinA.setPulseWidth(300);
 });
 
-pinA = pwm.open({channel: pins.IO3, period: 3, pulseWidth: 1});
+pinA = pwm.open({ channel: pins.IO3, period: 3, pulseWidth: 1 });
 assert(pinA != null && typeof pinA == "object",
```
Comment (jimmy-huang) [inline tests/test-pwm.js]: use !== and ===, this is not related to this patch, but you might want to fix these here.

File: tests/test-pwm.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ msTimer = setInterval(function () {
     msCount = msCount + 1;
 }, 50);
 
-setTimeout(function() {
+setTimeout(function () {
     assert(msTrue == 6 && msFalse == 14 && msCount == 20,
```
Comment (jimmy-huang) [inline tests/test-pwm.js]: same here

File: tests/test-pwm.js [hunk 3]
```diff
@@ -106,7 +106,7 @@ setTimeout(function() {
     cycleTimer = setInterval(function () {
        Flag = pinB.read();
 
-       if(Flag == oldFlag){
+       if (Flag == oldFlag) {
```
Comment (jimmy-huang) [inline tests/test-pwm.js]: same here

File: tests/test-pwm.js [hunk 4]
```diff
@@ -132,7 +132,7 @@ setTimeout(function() {
            }
        }
     }, 10);
-}, 1000);
+}, 1040);
```
Comment (jimmy-huang) [inline tests/test-pwm.js]: why is 1040 here? can you explain?

Comment (cuiyanx) [inline tests/test-pwm.js]: The msTimer is detection pulse, so we must Keep enough time to check it.
       

>  1000=50X20 < total-time < 50X21=1050

I have got 1040 for robustness of this case.

Comment (jimmy-huang) [inline tests/test-pwm.js]: ok, good to know, thanks.

**Pull request merged**